package.json

```diff
@@ -74,25 +73,25 @@
     "markdownlint": "^0.6.0",
     "markdownlint-cli": "^0.3.1",
     "marked": "^0.3.7",
+    "mini-css-extract-plugin": "^0.2.0",
     "mkdirp": "^0.5.1",
     "modularscale-sass": "^3.0.3",
     "moment": "^2.20.1",
     "ncp": "^2.0.0",
     "node-sass": "^4.5.3",
     "npm-run-all": "^4.1.1",
-    "postcss-loader": "^2.0.6",
+    "postcss-loader": "^2.1.3",
     "prism-languages": "^0.3.3",
     "prismjs": "^1.9.0",
     "raw-loader": "^0.5.1",
     "request": "^2.81.0",
     "sass-loader": "^6.0.6",
     "sitemap-static": "^0.4.2",
-    "style-loader": "^0.18.2",
     "tap-min": "^1.2.1",
     "tap-parser": "^6.0.1",
     "through2": "^2.0.3",
     "url-loader": "^0.5.9",
-    "webpack": "^3.10.0",
+    "webpack": "^4.2.0",
     "webpack-dev-server": "^2.9.7",
     "webpack-merge": "^4.1.0",
```

Comment:
i did not remove merge yet just in case, if no objections ill remove it after next update with target branch